.gitmodules

```diff
@@ -64,4 +64,7 @@
 	path = src/tools/clang
 	url = https://github.com/rust-lang-nursery/clang.git
 	branch = rust-release-80-v1
-  
\ No newline at end of file
+[submodule "src/polly"]
+	path = src/polly
+	url = https://github.com/llvm-mirror/polly.git
```

Comment:
Perhaps `rust-lang` should have its own fork of this repo?
I could imagine that if upstream polly gets synced with upstream llvm but rustc uses rust-lang-llvm with llvm upstream polly, there might be conflicts between let's say polly 8.0 and llvm 7.0 (the rustc-lang fork).

Reply:
We could track the `release_70` branch instead of `master` if version incompatibility is the only concern.